mods/ra/weapons/explosions.yaml

```
@@ -142,15 +142,38 @@ CivPanicExplosion:
 		Delay: 1
 
 BarrelExplode:
+	Warhead@Cluster: FireCluster
+		Weapon: BarrelCluster
+		Dimensions: 3,3
+		Footprint: _X_ X_X _X_
```

Can we change this to
```suggestion
		Footprint: _X_ XXX _X_
```
and then remove the other warheads from `BarrelExplode` and only use `BarrelCluster` (assuming we want to have a `LeaveSmudge` in the middle)?

---

Looking at video of OG, the barrel cell has different effects than the surrounding four fireballs. I actually need the barrel itself to have a crater instead of scorch smudge. I also need the barrel explosion to be `building` explode instead of `large_napalm`. 

Will add the crater smudge tonight.

---

Ah I missed that, then we should keep it more or less as-is. (And just add the smudge if needed.)